src/Factories/ComponentFactory.ts

```diff
@@ -65,17 +65,19 @@ export function create(fileName: string, node: ts.Node, checker: ts.TypeChecker)
         componentComposites.push(EnumFactory.create(enumSymbol));
 
         return;
-      } else if (childNode.kind === ts.SyntaxKind.FunctionDeclaration) {
-        const currentNode: ts.FunctionDeclaration = <ts.FunctionDeclaration>childNode;
-        if (currentNode.name === undefined) {
-          return;
-        }
-        const functionSymbol: ts.Symbol | undefined = checker.getSymbolAtLocation(currentNode.name);
-        if (functionSymbol === undefined) {
-          return;
-        }
-        componentComposites.push(MethodFactory.create(functionSymbol, currentNode, checker));
       }
+      // Functions are not methods in PlantUML! The (fake) methods (that are functions) get placed in the PlantUML outside of a class, and it won't render.
```

Comment:
Not quite following the problem. Is there example input and tsplant output that doesn't render? What doesn't render the output?

Reply:
Take a typescript file with no class that exports a function. The function is modeled as a method. The result in PlantUML will crash.

Reply:
I'm trying to find a simple example to reproduce the problem. I put an image in the PR that shows how it makes PlantUML crash. In that case, `indexOfSplitArray` is defined here: https://github.com/fuhrmanator/FamixTypeScriptImporter/blob/62191c407829e2a00f72886df3bf4c85298299cf/src/famix_functions/famix_functions_index.ts#L99

That function is defined in a module (not in a class). First, functions are not methods in UML (in principle, a TypeScript function can't be represented in a UML class diagram). Second, this function is put in a PlantUML outside of a class (because it's not defined in a class). This makes PlantUML crash with a syntax error. Perhaps I'm misunderstanding how you wanted that to work?

I will try to find a simple way to reproduce the problem.

Reply:
Here's a simple example:

**mySample.ts**
```typescript
export class Foo {
  a(): void {}
}
// a function is not a method in UML
export function f () {
  return "Hello, world!";
}
```

```bash
tplant -i mySample.ts
@startuml
class Foo {
    +a(): void
}
+f(): string
@enduml
```

The `+f(): string` should not be in the PlantUML (a function is not a method). Furthermore, since it's outside a class, it crashes PlantUML with a syntax error.

This bug didn't exist in 3.1.0 (as far as I can tell), and I believe it's caused by the code in the factory that I commented out. Hope this is clear now.

Reply:
FYI I use tplant in a GitHub action to draw the diagram of the API at https://github.com/fuhrmanator/FamixTypeScriptImporter?tab=readme-ov-file#typescript-metamodel-api-documentation-visualization 

I set the Action to use `tplant` 3.1.0 and the problem with the syntax error in PlantUML doesn't happen anymore.